aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,49 @@
 module aptos_framework::transaction_context {
+
+    use std::features;
+
+    /// AUID feature is not supported.
+    const EAUID_NOT_SUPPORTED: u64 = 3;
```

Comment:
Why is this starting at 3? Shouldnt this be 1?

Reply:
I actually used 1 initially. After I changed the variable name a few types, I think the linter automatically bumped up the number. Changed it back to 1.